lib-ng/mapper/geofence/beacon.js

```diff
@@ -0,0 +1,24 @@
+/* jshint ignore:start */
+var Alloy = require('alloy'),
+    _ = require('alloy/underscore')._,
+    Backbone = require('alloy/backbone');
+/* jshint ignore:end */
+
+/**
+ * Public functions
+ */
+
+function map(geofenceRegion) {
+    const beaconRaw = geofenceRegion.identifier.split('|');
+
+    return {
```

Comment:
No indeed, when you check the other mappers, it also doesn't contain a `beacon_id` ;)